main.go

```diff
@@ -57,6 +83,15 @@ func run(l *slog.Logger) error {
 	l.Info("building server")
 	s := NewServer(l, ar, nsl)
 
+	// configure TLS
+	if enableTLS {
+		s.WithTLS(true)
+		tlsOpts := []func(*tls.Config){
+			loadTLSCert(l, tlsCertificatePath, tlsCertificateKeyPath),
+		}
+		s.WithTLSOpts(tlsOpts)
```

Comment:
This seems redundant to me


```suggestion
		tlsOpts := []func(*tls.Config){
			loadTLSCert(l, tlsCertificatePath, tlsCertificateKeyPath),
		}
		s.WithTLSOpts(tlsOpts)
```

Reply:
Fixed in f9ea3c6 by making it variadic.